### ISSUE TITLE
column width for column with rotate text

### ISSUE THREAD

**Issue by neuwirthe** [davidgohel/flextable#159]
I am using flextable 0.5.5 on a Mac in Rstudio.
I want the first column in a table  containing rotated text narrower,
but using the width function does not work.

```
library(dplyr)
library(flextable)
my_df <- tibble(
  x = rep("head_col",5),
  num_col = 1:5 
)

my_df %>%
  flextable() %>%
  rotate(1:5,1,rotation="btlr",part="body") %>% 
  merge_v(1) %>%
  width(1,0.1)
```

Making the column wider works:

```
my_df %>%
  flextable() %>%
  rotate(1:5,1,rotation="btlr",part="body") %>% 
  merge_v(1) %>%
  width(1,0.1)
```

but changes the vertical location of the rotated text.

**Comment by davidgohel** [comment]
Could you check it is fixed with work done on #157 (they are related issues)?

**Comment by davidgohel** [comment]
I think I solved issues about text rotation - I will re-open that issue if necessary

KR
